packages/playground/_stories/main/Switch/Switch.stories.ts

```diff
@@ -60,4 +62,43 @@ export const Design = Template.bind({});
 Design.args = {
 	design: SwitchDesign.Graphical,
 	accessibleName: "graphical",
-};
\ No newline at end of file
+};
+
+export const RequiredInForm = Template.bind({});
+RequiredInForm.args = {
+	required: true,
+	name: "mySwitch",
```

Comment:
"conditions" sound like a better name for the server to receive from the from submit.